src/mdv/Arguments.cs

```diff
@@ -113,5 +127,37 @@ private static bool ParseFlagArg(string[] args, string name, bool defaultValue)
             arg.Equals(offStr, StringComparison.OrdinalIgnoreCase) ? false :
             value);
     }
+
+    public static bool TryInferInputsFromDirectory(string dir, out string path, out ImmutableArray<(string, string)> deltas)
+    {
+        var inputs = s_peExtensions.SelectMany(extension => Directory.GetFiles(dir, "*" + extension, SearchOption.TopDirectoryOnly)).ToArray();
+        if (inputs.Length != 1)
+        {
+            path = null;
+            deltas = ImmutableArray<(string, string)>.Empty;
+            return false;
+        }
+
+        var deltasBuilder = ImmutableArray.CreateBuilder<(string, string)>();
+        var i = 1;
+        while (true)
+        {
+            var pathWithoutExtension = System.IO.Path.Combine(dir, i.ToString());
+            var ilPath = pathWithoutExtension + ".il";
+            var mdPath = pathWithoutExtension + ".md";
```

Comment:
Before this gets encased in code forever, would it be better to use `.metadata`? 8.3 filenames are no longer a thing, and `.md` are pretty well known as markdown files.

I know its not an issue here, and its unlikely anyone would think to run mdv on a random markdown file, but thought I'd mention it.

Reply:
Hmm, good point but our ENC logger already uses ".md". I can change that though.